src/test/injecture-test.js

```diff
@@ -186,8 +186,8 @@ describe('Injecture', function() {
 
   it('can add reducers to supply logic how get the right interface', (done) => {
 
-    injecture.addReducers({
-      key: 'bar',
```

Comment:
This is technically an "API" change, I felt `interfaceType` is much more accurate as `key` is NOT a class key at all.